src/components/TagFilters.svelte

```diff
@@ -13,24 +14,40 @@
     ['metric tutorials', 'Metric Tutorials'],
   ]
 
-  $: ({ tags, onlyPro } = $page.query)
-  $: active = onlyPro !== undefined ? 'pro' : (tags || '').toLowerCase()
+  $: ({ tags, onlyPro, isPulse } = $page.query)
+  $: active = (tags, onlyPro, isPulse, setActive())
+  $: isMobile = $session.isMobile
+
+  function setActive() {
+    if (onlyPro !== undefined) return 'pro'
+    if (isPulse !== undefined) return 'pulse'
+
+    return (tags || '').toLowerCase()
+  }
 </script>
 
-<div class="row c-waterloo mrg-xl mrg--t mrg--b">
-  {#each TAGS as [link, label]}
+<div class="row c-waterloo {isMobile ? 'body-2 nowrap' : 'mrg-xl'} mrg--t mrg--b">
+  {#each TAGS as [link, label], idx}
     {@const href = link ? `${base}?tags=${link}` : base}
-    <a {href} class="btn-2" class:active={active === link}>{label}</a>
+    {#if idx === 0 && isMobile}
+      <a href="/" class="btn-2" class:active={active === link}>{label}</a>
+      <a href="?isPulse" class="btn-2" class:active={active === 'pulse'}>Pulse</a>
+      <a href="?onlyPro" class="pro btn-2 btn-1 btn--orange" class:active={active === 'pro'}>
+        Only for PRO
+      </a>
+    {:else}
+      <a {href} class="btn-2" class:active={active === link}>{label}</a>
+    {/if}
   {/each}
```

Comment:
Do 2 renders based on the `isMobile` variable. Inside `isMobile` branch iterate over `TAGS.slice(1)`